torch_xla/core/dynamo_bridge.py

```diff
@@ -217,6 +241,9 @@ def extract_compiled_graph(xla_model: torch.fx.GraphModule, xla_args):
       tensor_id: i for i, tensor_id in enumerate(args_tensor_ids)
   }
 
+  # partition the graph for unsupported ops
+  xla_model = maybe_partition_graph(xla_model)
```

Comment:
now `xla_model` becomes a `partitioned_graph`, and what happen next? In below code we will trace through this model `xla_out = xla_model(*xla_args)`, are we going to get a single graph and a single hash?